src/test/config.sol

```diff
@@ -109,7 +110,7 @@ contract Config {
         // Values for all system configuration changes
         //
         afterSpell = SystemValues({
-            line_offset:           500 * MILLION,           // Offset between the global line against the sum of local lines
+            line_offset:           502 * MILLION,           // Offset between the global line against the sum of local lines
```

Comment:
So we needed to add to the offset because this ilk type is not in the registry, so the lines can't be summed.
Now a part from this issue, I think we should add it to the registry, maybe with a new type class? cc @brianmcmichael